1-exercises/A-array-find/exercise.js

```diff
@@ -16,8 +16,16 @@ let names = [
   "Karim",
   "Ahmed",
 ];
+function findLongNameThatStartsWithA(arr) {
+  for (let i of arr) {
+    if (i[0] === "A") {
+      return arr.length > 7;
+    }
+  }
+}
 
-let longNameThatStartsWithA = findLongNameThatStartsWithA(names);
+
+let longNameThatStartsWithA = names.find(findLongNameThatStartsWithA);
```

Comment:
Great job Elahe,
It's good to know that there is shorter way in order to use the array methods.
Here is the other way of solving this exercise:

**function findLongNameThatStartsWithA(arr) {
  return arr.find( name => name.length > 7 )
}**